modules/client/src/tokenVoting/interfaces.ts

```diff
@@ -261,3 +264,38 @@ export enum UnwrapTokensStep {
 export type UnwrapTokensStepValue =
   | { key: UnwrapTokensStep.UNWRAPPING; txHash: string }
   | { key: UnwrapTokensStep.DONE };
+
+export type DelegateTokensParams = {
+  tokenAddress: string;
+  delegatee: string;
+};
+
+export enum DelegateTokensStep {
+  DELEGATING = "delegating",
+  DONE = "done",
+}
+export type DelegateTokensStepValue = {
+  key: DelegateTokensStep.DELEGATING;
+  txHash: string;
+} | { key: DelegateTokensStep.DONE };
+
+export type SubgraphTokenVotingMember = {
+  address: string;
+  balance: string;
+  votingPower: string;
+  delegatee: {
+    address: string;
+  };
+  delegators: {
+    address: string;
+    balance: string;
+  }[];
+};
+
+export type TokenVotingMember = {
+  address: string;
+  balance: bigint;
+  votingPower: bigint;
+  delegatee: string | null;
```

Comment:
If it can be null, then I would add a simple example above for someone who just doesn't delegate anything.